scan.go

```diff
@@ -97,6 +98,26 @@ func (s *ScanService) SearchSource(searchSource *SearchSource) *ScanService {
 	return s
 }
 
+// Source allows the user to set the request body manually without using
+// any of the structs and interfaces in Elastic.
+func (s *ScanService) Source(sourceToSet interface{}) *ScanService {
```

Comment:
Ok, as mentioned this is following the convention set in SearchSource. I agree with Body(); the function should take either a valid JSON string, or a valid JSON map.